libraries/lib-time-and-pitch/StaffPad/TimeAndPitch.cpp

```diff
@@ -376,9 +376,10 @@ void TimeAndPitch::_process_hop(int hop_a, int hop_s)
        // is what one typically gets at those frequencies.
        const int imagingBeginBin = fftSize / 2 * _pitchFactor + 1;
        const auto n = _numBins - imagingBeginBin;
-       vo::rotate(
```

Comment:
We spent a lot of time and effort to optimize it, it is weird to see it go.

Reply:
You go from SIMD to scalar algorithm, and I don't see any good reason for that

Reply:
This removal is only the removal of an earlier commit in this very PR ; the older code is left untouched.
Bad things happen if I use `vo::rotate` because it expects aligned float pointers, and `pSpec + imagingBeginBin` isn't necessarily aligned.